### PR TITLE
Add UI synching funciton for published happs to `hosted-happ-monitor`

### PR DESCRIPTION
**Pull request by peeech**
`hosted-happ-monitor` runs every 15 min to sync hha data with KV store. This PR adds functionality that allows script to update state of UIs based on happs' status in hha:
- checks if UI is published to S3
- checks what version of UI is published
- if versions mismatch or UI not present removes stale UI and uploads current one

The solution like this will not scale in the future, because with increasing number of hosted happs the number of checks becomes a bottlenceck. Better solution for future releases is still TBD.

User flow for happ publishing: [here](https://miro.com/app/board/o9J_lKCrKpg=/?moveToWidget=3458764515796198581&cot=14)

### CODE REVIEW

**Comment by alastairong** [comment]
@peeech Oops I misread this PR and bumped it for latest develop!

File: overlays/holo-nixpkgs/hosted-happ-monitor/src/index.js [hunk 1]
```diff
@@ -43,11 +43,18 @@ const main = async () => {
     payload: null
   })
 
+  // Loop through all the happs and check their UI status
+  // This is where an idea of hosted-happ-monitor fails at the scale
+  // because for each hosted happ present in HHA we need to make a `curl` call
+  const syncUiPromises = happs.map(happ => syncUiState(happ))
```
Comment (zo-el) [inline overlays/holo-nixpkgs/hosted-happ-monitor/src/index.js:49]: This list of happs are only the published happs. You will need to remove all the UI's that were disabled i.e. not in this list but are hosted on S3

**Comment by peeech** [comment]
This PR is very much outdated, not WIP anymore

**Comment by alastairong** [comment]
Ok. Let's make sure not to delete the branch though.

**Pull request closed without merging**